OpenRA.Game/Effects/FlashTarget.cs

```diff
@@ -25,7 +25,7 @@ public FlashTarget(Actor target, Player asPlayer = null, int ticks = 4)
 			this.target = target;
 			player = asPlayer;
 			remainingTicks = ticks;
-			foreach (var e in target.World.Effects.OfType<FlashTarget>().Where(a => a.target == target).ToArray())
+			foreach (var e in target.World.Effects.OfType<FlashTarget>().Where(a => a.target == target).ToList())
```

Comment:
>Short lived variables use ToList. This is because ToArray requires an extra copy to output the final size.

My question is why explicitly enumerate this at all?